Documentation/cluster-crd.md

```diff
@@ -23,7 +23,8 @@ Settings can be specified at the global level to apply to the cluster as a whole
 - `versionTag`: The version (tag) of the `rook/rook` container that will be deployed. Upgrades are not yet supported if this setting is updated for an existing cluster, but upgrades will be coming.
 - `dataDirHostPath`: The host path where config and data should be stored for each of the services. If the directory does not exist, it will be created. Because this directory persists on the host, it will remain after pods are deleted.  Therefore, for test scenarios, the path must be deleted if you are going to delete a cluster and start a new cluster on the same hosts.  More details can be found in the Kubernetes [host path docs](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath).
 If this value is empty, each pod will get an ephemeral directory to store their config files that is tied to the lifetime of the pod running on that node. More details can be found in the Kubernetes [empty dir docs](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir).
-- `hostNetwork`: uses host network instead of using the SDN below the containers
+- `hostNetwork`: uses network of the hosts instead of using the SDN below the containers.
+- `monSize`: set the amount of mons to be started (default: `3`).
```

Comment:
could you name it `monCount`? I know it was called `size` internally, but a user-visible name seems better to be `count`.

Reply:
If you change this value in the cluster.yaml and apply it, will the update trigger adding/removing mons? That might be a separate PR, but would be good to have that.

Reply:
@travisn `onUpdate` isn't implemented yet (see https://github.com/rook/rook/blob/master/pkg/operator/cluster/controller.go#L155).
I'm creating an issue for updating the cluster CRD. I also think there needs to be further discussion in that issue what should be allowed to be updated.